conda_lock/conda_lock.py

```diff
@@ -165,21 +165,27 @@ def print_proc(proc):
     try:
         proc.check_returncode()
     except subprocess.CalledProcessError:
-        try:
-            err_json = json.loads(proc.stdout)
-            message = err_json["message"]
-        except json.JSONDecodeError as e:
-            print(f"Failed to parse json, {e}")
-            message = ""
-
-        print(f"Could not perform conda install using {file} lock file into {prefix}")
-        if message:
-            print(message)
+        print(f"Could not perform conda install using {file} lock file into {name or prefix}")
```

Comment:
Just a small fix. Otherwise it says "[..] lock file into None", when you use `name` instead of `prefix`.